collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh

```diff
@@ -244,16 +244,6 @@ networking() {
   mkdir -p $TMPDIR/networking
   iptables-save > $TMPDIR/networking/iptablessave 2>&1
   ip6tables-save > $TMPDIR/networking/ip6tablessave 2>&1
-  if [ ! "${OSRELEASE}" = "sles" ]
-    then
-      IPTABLES_FLAGS="--wait 1"
-  fi
```

Comment:
We probably need these lines.

Reply:
IPTABLES_FLAGS are only used in those 6 commands which I removed. other than that, it is not being used else where